V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayVpnService.kt

```diff
@@ -549,4 +550,3 @@ class V2RayVpnService : VpnService() {
         }
     }
 }
-
```

Comment:
This is trivial but I just want to mention I've seen projects having issue in Linux without line breaks at the end of files.
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
I think L552 is just a blank line... And I'm sure L551 have `\n`